DESCRIPTION

```diff
@@ -62,7 +62,6 @@ Collate:
     'utilities-pk-analyses.R'
     'utilities-population.R'
     'utilities-quantity.R'
-    'utilities-results.R'

```

Comment:
There was a utilities-results but you had also created a utilties-simulation-results.
I moved everything into one